streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignor.java

```diff
@@ -709,10 +734,17 @@ public Subscription subscription(final Set<String> topics) {
                 }
 
                 // finally, encode the assignment before sending back to coordinator
+
+                final ByteBuffer bb;
+                if (minUserMetadataVersion >= 4)
+                    bb = new AssignmentInfo(minUserMetadataVersion, tasksByHostState, active, standby, 0).encode();
+                else
+                    bb = new AssignmentInfo(minUserMetadataVersion, active, standby, partitionsByHostState, 0).encode();
+                assignmentSize = assignmentSize + bb.array().length;
                 assignment.put(consumer, new Assignment(
-                    activePartitions,
-                    new AssignmentInfo(minUserMetadataVersion, active, standby, partitionsByHostState, 0).encode()));
+                    activePartitions, bb));
             }
+            log.info("Size of assignment is: " + assignmentSize + " bytes. Total hosts in this assignment are " + clientsMetadata.size());
```

Comment:
Should it be total instances/processes instead of hosts?

Reply:
Yes, I will make the change.